docs/docs/getting-started/ssl.md

```diff
@@ -1,50 +1,20 @@
 ---
 title: SSL Configuration
-layout: article
-category: install
+sidebar_position: 4
 ---
 
 To increase the security of your server, you can enable encrypted access to the LibreTime administration interface, and direct your users towards this more secure login page. The main advantage of using this encryption is that your remote users' login names and passwords are not sent in plain text across the public Internet or untrusted local networks, such as shared Wi-Fi access points.
 
-### Deploying a certificate with Certbot
+## Deploying a certificate with Certbot (Recommended)
 
 One of the fastest, easiest, and cheapest ways to get an SSL certificate is through [Certbot](https://certbot.eff.org/), as created by the
-Electronic Frontier Foundation. There are some requirements for this process:
+Electronic Frontier Foundation. To use Certbot, your Libretime installation must be open to the internet on port 80.
 
-- you have an HTTP website (already installed and configured by default by the LibreTime installer) and
-- this website is open to the public internet (likely via. port forwarding if your computer is behind a firewall) and
-- the server is accessible to the public via. port 80
+Follow [Certbot's documentation](https://certbot.eff.org/instructions) for your OS and webserver to install an SSL certificate. You'll need to renew the certificate every 90 days to keep your installation secure.
 
-If you aren't able to verify all three requirements, you may want to try a self-signed certificate (see next section).
+If you aren't able to use Certbot, you may want to try a self-signed certificate.
 
-These instructions come from Certbot's website and assume that you are using an Apache web server
-running on Ubuntu 18.04 LTS (the Apache web server is installed with LibreTime by default).
-Instructions for other Debian-based OSes are similar, but check with Certbot for clarification.
-
-Note: all instructions require you to have sudo privileges
-
-First, add Certbot's PPA using:
-
-```
-    sudo apt-get update
-    sudo apt-get install software-properties-common
-    sudo add-apt-repository universe
-    sudo add-apt-repository ppa:certbot/certbot
-    sudo apt-get update
-```
-
-Next, install Certbot and install the SSL certificate using the below commands:
-
-```
-sudo apt-get install certbot python3-certbot-apache
-sudo certbot --apache  # get and install the certificate
-sudo certbot certonly --apache  # to only get the certificate, not install it using Certbot
-```
-
-You can test certificate renewal by running `sudo certbot renew --dry-run`.
-Head to your server's IP address to check to see that the installation worked.
```

Comment:
I also think we should not rewrite documentation for other software, certbot has an good documentation let's delegate that to them.